src/azul/indexer/index_service.py

```diff
@@ -265,6 +265,7 @@ def transform(self,
                        of additions.
         """
         plugin = self.metadata_plugin(catalog)
+        bundle.reject_if_contains(config.manifest_column_joiner)
```

Comment:
```suggestion
        bundle.reject_if_contains()
```

Needless parameterization.